specification/columns/skupriceid.md

```diff
@@ -2,29 +2,29 @@
 
 A SKU Price ID is a unique identifier that defines the unit price used to calculate the charge. SKU Price ID can be referenced on a [*price list*](#glossary:price-list) published by a provider to look up detailed information, including a corresponding list unit price. The composition of the properties associated with the SKU Price ID may differ across providers. SKU Price ID is commonly used for analyzing cost based on pricing properties such as Terms and Tiers.
 
-The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SkuPriceId in the provider published *price list*. The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is "Purchase" or "Usage". SkuPriceId MUST NOT be null when [ChargeSubcategory](#chargesubcategory) is "Refund" and the refund is related to charges with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of commitment discount flexibility.
+The SkuPriceId column MUST be present in the billing data when the provider publishes a SKU price list. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SkuPriceId in the provider-published *price list*. The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is "Purchase" or "Usage". SkuPriceId MUST NOT be null when [ChargeSubcategory](#chargesubcategory) is "Refund" and the refund is related to charges with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of commitment discount flexibility.
```

Comment:
Similar to SkuId, shouldn't this be required whenever a SKU can support multiple prices?
```suggestion
The SkuPriceId column MUST be present in the billing data when the provider has any [SkuId](#skuid) that can have multiple prices for different variations of the SKU. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SkuPriceId in the provider-published *price list*. The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is "Purchase" or "Usage". SkuPriceId MUST NOT be null when [ChargeSubcategory](#chargesubcategory) is "Refund" and the refund is related to charges with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of commitment discount flexibility.
```

Reply:
As I recall, SkuId relies on SkuPriceId and if there are no multiple prices SkuId matches the skuPriceId. 
 Both are present or not, depending on the SKU Price list being published.

One more question - ListUnitPrice is not necessarily published (one of the latest conversations), but the SkuPriceId  is still present if SKU Price list is published - we are not changing that, right?

Reply:
Same reasoning as the other instance. Keeping it at if the provider publishes gives some confidence that if it's relevant, the provider will provide this which then requires it to be available in the billing data - regardless of if there's one or more.